### PR TITLE
Debug: Changed ImageBase field to load address for debug info search.

### PR DESCRIPTION
**Pull request by MikhailKrichanov**

### CODE REVIEW

**Comment by mikebeaton** [comment]
Thank you for the quick fix - I can't test before this eve., but will do then.

File: Debug/Scripts/gdb_uefi.py [hunk 1]
```diff
@@ -362,7 +363,7 @@ def parse_edii(self, edii, count):
                 entry = entry['NormalImage']
                 self.parse_image(entry['LoadedImageProtocolInstance'], syms)
             else:
-                print(f"Skipping unknown EFI_DEBUG_IMAGE_INFO(Type 0x{entry['ImageInfoType'].dereference():x})")
+                print("Skipping unknown EFI_DEBUG_IMAGE_INFO(Type " + str(entry['ImageInfoType'].dereference()) + ")")
```
Comment (dreamwhite) [inline Debug/Scripts/gdb_uefi.py:366]: I think the print statement can be replaced by:

```python
print(f"Skipping unknown EFI_DEBUG_IMAGE_INFO(Type {str(entry['ImageInfoType'].dereference())})"
```

File: Debug/Scripts/lldb_uefi.py [hunk 2]
```diff
@@ -373,7 +374,7 @@ def parse_edii(self, edii, count):
                 entry = entry.GetChildMemberWithName('NormalImage')
                 self.parse_image(entry.GetChildMemberWithName('LoadedImageProtocolInstance'), syms)
             else:
-                print(f'Skipping unknown EFI_DEBUG_IMAGE_INFO (Type 0x{image_type:x})')
+                print('Skipping unknown EFI_DEBUG_IMAGE_INFO (Type ' + str(image_type) + ')')
```
Comment (dreamwhite) [inline Debug/Scripts/lldb_uefi.py:377]: Same here:

```python
print(f'Skipping unknown EFI_DEBUG_IMAGE_INFO (Type {str(image_type)})')
```

**Review by dreamwhite — CHANGES_REQUESTED** [review]
Some small hints for Python3-like code

**Comment by mikebeaton** [comment]
Correct me if I'm wrong, but even with the above fixes, I don't think I can now step out of the OpenCore image and into the OVMF image (e.g. if both were built using NOOPT), as I could before the new build system.

I'm not sure if everything built with normal EDK-II build commands (e.g. using the EDK-II provided `build.sh` script in the `OvmfPkg` directory) in theory should also use new build commands and new loaders, and so should load debug info correctly with these `efidebug.toold` changes, or not?

**Pull request merged**